.travis.yml

```
@@ -11,7 +11,7 @@ env:
     - SCVERSION="stable"
 
 before_install:
-    - docker pull quay.io/coreos-assembler/coreos-assembler:latest
+    - docker pull registry.fedoraproject.org/fedora:31
     - docker build -t coreos-assembler-test -f Dockerfile.dev .
```

Hmm, I think what we actually want here is just

```suggestion
    - docker build -t coreos-assembler-test -f Dockerfile .
```

right? The `Dockerfile` already contains the right `FROM` image to use.